modules/admin-ui-picard/app/src/thunks/statisticsThunks.js

```diff
@@ -0,0 +1,226 @@
+import axios from "axios";
+import moment from "moment";
+import {createChartOptions, createDownloadUrl} from "../utils/statisticsUtils";
+import {getHttpHeaders} from "../utils/resourceUtils";
+import {logger} from "../utils/logger";
+import {getStatistics} from "../selectors/statisticsSelectors";
+import {
+    loadStatisticsFailure,
+    loadStatisticsInProgress,
+    loadStatisticsSuccess, setOrganizationId,
+    updateStatisticsFailure, updateStatisticsSuccess
+} from "../actions/statisticsActions";
+
+/* thunks for fetching statistics data */
+
+export const fetchStatisticsPageStatistics = () => async (dispatch) => {
+    // get organization id from API, then get statistics for that organization
+    axios.get('/info/me.json')
+        .then( response => {
+            const organizationId = response.data.org.id;
+
+            dispatch(setOrganizationId(organizationId));
+
+            dispatch(fetchStatistics(organizationId, 'organization', getStatistics, loadStatisticsInProgress,
+                loadStatisticsSuccess, loadStatisticsFailure));
+        })
```

Comment:
This information is already saved in the redux state. Take a look at the user information reducer.